packages/xo-server/sample.config.toml

```diff
@@ -52,6 +52,11 @@
 
 #=====================================================================
 
+# Configuration for backupNg
+[backupNgOptions]
+# timeout in milliseconds
+#timeout = 2e3
+
```

Comment:
Advanced/internal config are not documented in `sample.config.toml`, only entries that we expect users to customize are :-).